alf/bin/play.py

```diff
@@ -53,9 +53,6 @@ def _define_flags():
     flags.DEFINE_float('epsilon_greedy', 0., "probability of sampling action.")
     flags.DEFINE_integer('random_seed', None, "random seed")
     flags.DEFINE_integer('num_episodes', 10, "number of episodes to play")
-    flags.DEFINE_integer('max_episode_length', 0,
```

Comment:
This flag is actually useful in some cases, e.g. when you only want to render a short video clips.

Reply:
You can just change the conf file?